org.zaproxy.ZAP.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "org.zaproxy.ZAP",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "zap",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11" 
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--env=JAVA_HOME=/app/jre",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [ "/usr/lib/sdk/openjdk11/install.sh" ]
+        },
+        {
+            "name": "zap",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/bin",
+                "ln -s /app/jre/bin/java /app/bin/java",
+                "install -Dm644 org.zaproxy.ZAP.png /app/share/icons/hicolor/128x128/apps/org.zaproxy.ZAP.png",
+                "install -Dm644 org.zaproxy.ZAP.desktop /app/share/applications/org.zaproxy.ZAP.desktop",
+                "install -Dm644 org.zaproxy.ZAP.appdata.xml /app/share/appdata/org.zaproxy.ZAP.appdata.xml",
+                "mkdir /app/share/owasp-zap",
+                "rm org.zaproxy.ZAP.png org.zaproxy.ZAP.desktop org.zaproxy.ZAP.appdata.xml",
+                "cp -r ./* /app/share/owasp-zap",
+                "ln -s /app/share/owasp-zap/zap.sh /app/bin/zap"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/zaproxy/zaproxy/releases/download/v2.8.0/ZAP_2.8.0_Linux.tar.gz",
```

Comment:
It could use 2.8.1 (it's the latest for Linux).

Reply:
consider it done.

Reply:
Thanks :)